app/components/reports/subject.hbs

```diff
@@ -31,6 +54,13 @@
     </div>
   </div>
   <div class="content-container">
+    <div class="description" data-test-report-description>
+      {{#if this.reportDescriptionLoaded}}
+        {{this.reportDescription}}
+      {{else}}
+        <LoadingSpinner />
```

Comment:
I'm not sure about the loading spinners here. Mostly they don't show up because the data for them is already loaded, but when they do it's so fast as to appear janky. Maybe call both `this.reporting.buildReportDescription(selectedReport)` and `this.reporting.buildReportTitle(selectedReport)` in the routes afterModel hook to ensure this data is loaded and then remove the loading indicators?

Reply:
:heavy_check_mark: